packages/debugger/style/icons.css

```
@@ -25,3 +25,13 @@ button.jp-Button.jp-mod-minimal.jp-TableView:hover {
 .jp-ViewModeSelected {
   background-color: var(--jp-layout-color2);
 }
+
+/* Pause on exceptions */
+button.jp-Button.bp3-button.bp3-minimal.jp-PauseOnExceptions:hover {
```

Blueprint has been removed

```suggestion
button.jp-Button.jp-mod-minimal.jp-PauseOnExceptions:hover {
```